src/doc/rustdoc/src/unstable-features.md

```diff
@@ -135,8 +135,20 @@ Book][unstable-masked] and [its tracking issue][issue-masked].
 ## Document primitives
 
 Since primitive types are defined in the compiler, there's no place to attach documentation
-attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way to generate
-documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way
+to generate documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+
+## Document keywords
+
+Keywords are documented in the standard library (look for "match" for example). However,
+you might write a macro or a proc-macro adding new keywords. If you want to document them,
+you can by using `#[doc(keyword)]` on an empty module. Example:
+
+```rust,ignore (needs nightly feature)
```

Comment:
Why is this ignored? Doctests are run on nightly Rust. Just add `#![feature(doc_keyword)]`.

Reply:
:+1: